### PR TITLE
Add DEMOpp macro files with ANODE grid

### PR DESCRIPTION
**Pull request by jmunozv**
This PR adds examples macro files for DEMOpp with a grid as ANODE.

### CODE REVIEW

File: macros/DEMOPP_grid.config.mac [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## ----------------------------------------------------------------------------
+## nexus | DEMOPP_plate.config.mac
```
Comment (paolafer) [inline macros/DEMOPP_grid.config.mac]: Modify the name accordingly.

File: macros/DEMOPP_grid.config.mac [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+##
+## Configuration macro to simulate electrons in the DEMO++ geometry
+## with a quartz plate as anode.
```
Comment (paolafer) [inline macros/DEMOPP_grid.config.mac]: Modify the description accordingly.

File: macros/DEMOPP_grid.init.mac [hunk 3]
```diff
@@ -0,0 +1,25 @@
+## ----------------------------------------------------------------------------
+## nexus | DEMOPP_plate.init.mac
```
Comment (paolafer) [inline macros/DEMOPP_grid.init.mac]: Modify as above.

**Review by paolafer — CHANGES_REQUESTED** [review]

**Comment by jmunozv** [comment]
All descriptions fixed

**Review by jmunozv — COMMENTED** [review]
All descriptions fixed

**Review by paolafer — APPROVED** [review]
The example shows the usage of a DEMO geometry with the mesh instead of the quartz plate. The tests run, therefore I approve this PR.

**Pull request merged**